### PR TITLE
chore(php): updated dependencies to PHP 8.x and symfony/finder 6.0

### PR DESCRIPTION
**Pull request by schakko**

### CODE REVIEW

**Comment by PHLAK** [comment]
Thank you but I try to maintain support for all [currently supported PHP versions](https://www.php.net/supported-versions.php). This means support for PHP 7.4 is needed until at least January 2023.

That being said I don't believe there would be any breaking changes to require `symfony/finder` 5.X _or_ 6.X (though I'd have to test it to be sure).

File: composer.json [hunk 1]
```diff
@@ -20,8 +20,8 @@
         }
     ],
     "require": {
-        "php": ">=7.2",
-        "symfony/finder": "^5.0"
+        "php": ">=8.0",
+        "symfony/finder": "^6.0"
```
Comment (PHLAK) [inline composer.json]: Let's try requiring `symfony/finder` 5 _or_ 6 here and bumping the required PHP version to `7.4`.

```suggestion
        "php": ">=7.4",
        "symfony/finder": "^5.0 || ^6.0"
```

**Comment by schakko** [comment]
> Thank you but I try to maintain support for all [currently supported PHP versions](https://www.php.net/supported-versions.php). This means support for PHP 7.4 is needed until at least January 2023.
> 
> That being said I don't believe there would be any breaking changes to require `symfony/finder` 5.X _or_ 6.X (though I'd have to test it to be sure).

Yes, symfony/finder 5.X _or_ 6.X LGTM.

**Comment by PHLAK** [comment]
@schakko I've updated `master` to no longer test PHP 7.2 and 7.3. Please rebase this PR and I'll get it merged.

**Comment by schakko** [comment]
@PHLAK Sorry for the late reply, I've updated the PR accordingly.

**Comment by PHLAK** [comment]
Thanks @schakko! Just in time for the [PHP 8.2 release](https://www.php.net/archive/2022.php#2022-12-08-1) and [PHP 7.4 to reach end-of-life](https://www.php.net/supported-versions.php). :sweat_smile:

**Pull request merged**